roles/roles.tf

```
@@ -774,20 +800,197 @@ data "aws_iam_policy_document" "fargate_logging_policy" {
   }
 }
 
+# FARGATE [Common : Databricks and EMR]
 resource "aws_iam_policy" "fargate_logging" {
   count  = var.fargate_enabled ? 1 : 0
   name   = "tecton-${var.deployment_name}-fargate-logging-to-kinesis-firehose"
   policy = data.aws_iam_policy_document.fargate_logging_policy[0].json
 }
 
+# FARGATE [Common : Databricks and EMR]
 resource "aws_iam_role_policy_attachment" "logging" {
   count      = var.fargate_enabled ? 1 : 0
   role       = aws_iam_role.eks_fargate_pod_execution[0].name
   policy_arn = aws_iam_policy.fargate_logging[0].arn
 }
 
+# FARGATE [Common : Databricks and EMR]
 resource "aws_iam_role_policy_attachment" "fargate_pod_execution" {
   count      = var.fargate_enabled ? 1 : 0
   role       = aws_iam_role.eks_fargate_pod_execution[0].name
   policy_arn = "arn:aws:iam::aws:policy/AmazonEKSFargatePodExecutionRolePolicy"
 }
+
+########################################################################
+##### Below are all the resources brought up for satellite cluster #####
```

```suggestion
##### Satellite cluster #####
```
Does it make sense to just put this in a separate `satellite.tf` file?

---

I thought we discussed this offline, and it was ok to keep them in the same file. Do you want me to split into new file?